labgrid/driver/usbloader.py

```diff
@@ -87,7 +87,7 @@ def load(self, filename=None):
 
 @target_factory.reg_driver
 @attr.s(eq=False)
-class RKUSBDriver(Driver, BootstrapProtocol):
```

Comment:
We usually let the old `RKUSBDriver` raise a `DeprecationWarning` and instead use the new driver, i.e. see `NetworkUSBStorageDriver`.

Reply:
This won't work transparently, because I am changing the name of the tool key as well, but I can do this, so users are nudged into the right direction.